bedrock/firefox/templates/firefox/compare/chrome.html

```diff
@@ -0,0 +1,312 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('Six of the best browsers in direct comparison') }}{% endblock %}
+
+{% block page_desc %}
+{{ _('Find out how Firefox, Chrome, Edge, Safari, Opera and Internet Explorer differ in terms of privacy, utility and portability.') }}
+{% endblock %}
+
+{% block page_css %}
+{{ css_bundle('compare-details') }}
+{% endblock %}
+
+{% block content %}
+<section class="mzp-c-hero compare-details-hero compare-details-hero--chrome">
+  <div class="mzp-l-content l-compare-details-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('Compare Firefox Browser with Google Chrome') }}</h1>
+    </div>
+  </div>
+</section>
+
+<div class="mzp-l-content l-compare-details-content">
+  <article class="mzp-c-article">
+    <section class="compare-details-section-summary">
+        <p class="compare-details-section-intro">
+          {% trans %}
+          In 2008, Google introduced Chrome and its impact as an innovation in browser technology was immediately apparent. It was faster for loading sites, took up minimal screen space and offered an undeniably sleek user interface.
+          {% endtrans %}
+        </p>
+
+        <p>
+          {% trans %}
+          Fast-forward to today, the competitive landscape for browsers has changed with many people beginning to question just what is happening to their online data such as browsing history, passwords, and other sensitive information.  A lot has changed since 2008 when Chrome came onto the scene. Firefox has been heads down, working to redesign their interface, provide a growing number of privacy and performance enhancements and handy browser tools.
+          {% endtrans %}
+        </p>
+
+        <p>
+          {% trans %}
+          And so here we are, the browser-wars are escalating once again and it’s time to reevaluate and compare the Firefox Browser vs Google Chrome.
+          {% endtrans %}
+        </p>
+
+        <img src="{{ static('img/firefox/compare/compare-browser-chrome.png') }}" alt="" height="424" width="541" class="compare-details-image">
```

Comment:
@stephaniehobson hi stephanie! not sure if this is the right place to comment, but regarding the above - i typically export everything at x2, but going forward i can append these with high-res and provide the regular size image - since that seems to be the default flow for all these assets.